packages/cozy-client/src/models/file.js

```diff
@@ -115,6 +115,29 @@ export function normalize(file) {
   return { _id: id, id, _type: doctype, ...file }
 }
 
+/**
+ * Get the list of illegal characters in the file name
+ *
+ * Ensures existence of `_id` and `_type`
+ *
+ * @public
+ * @param {object} file - object representing the file
```

Comment:
import { IOCozyFile } from '../../types' (something like that)

and then use the `IOCozyFile` as type's param ;)

Reply:
we won't implement anything in the cozy-client package. This thread is outdated